realwork/WatchApp Extension/NotificationController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NotificationController.swift
+//  WatchApp Extension
+//
+//  Created by Nathan Ngobale on 2020/04/29.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import WatchKit
+import Foundation
+import UserNotifications
+
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Limit vertical whitespace to a single empty line. Currently 2.](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5444460)